rs/tests/nested/guestos_upgrade.rs

```diff
@@ -4,7 +4,7 @@ use std::time::Duration;
 
 fn main() -> Result<()> {
     SystemTestGroup::new()
-        .with_setup(|env| nested::config(env, 1, None))
+        .with_setup(nested::config)
```

Comment:
Any change `config` can be renamed to `setup`? (Note that "config" was the old way of naming setup function).
```suggestion
        .with_setup(nested::setup)
```